frontend/packages/kubevirt-plugin/src/components/virtualization-overview/running-vms-per-template-card/VMsChartLegendLabel.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+import './running-vms-per-template-card.scss';
+
+export const VMsChartLegendLabel = ({ item }) => {
+  const iconStyle = { color: item.color };
+
+  return (
+    <>
+      <i className="fas fa-square kv-running-vms-card__legend-label--color" style={iconStyle} />
+      <span className="kv-running-vms-card__legend-label--count">{item.vmCount}</span> {item.name}
```

Comment:
notes for improvments:
a - `item` can have a type.
b - You can add link:
`/k8s/ns/<NS>/kubevirt.io~v1~VirtualMachine?labels=vm.kubevirt.io/template=<template name>`

for example:
`/k8s/all-namespaces/kubevirt.io~v1~VirtualMachine?labels=vm.kubevirt.io/Ftemplate=windows10-desktop-medium`

Reply:
I didn't see that you had lgtm'd it. I made the suggested changes. PTAL